events/tests/importers/test_lippupiste.py

```diff
@@ -0,0 +1,71 @@
+import json
+from copy import deepcopy
+from pathlib import Path
+
+import pytest
+from django.conf import settings
+
+from events.importer.lippupiste import LippupisteImporter
+from events.models import Event
+from events.tests.factories import DataSourceFactory, KeywordFactory
+
+
+@pytest.fixture
+def importer():
+    importer = LippupisteImporter({"force": False})
+    importer.setup()
+    return importer
+
+
+@pytest.fixture(autouse=True)
+def tprek_datasource():
+    return DataSourceFactory(id="tprek")
+
+
+@pytest.fixture
+def yso_datasource():
+    return DataSourceFactory(id="yso")
+
+
+@pytest.fixture
+def drama_keyword(yso_datasource):
+    return KeywordFactory(id="yso:p2625", data_source=yso_datasource, name="Draama")
+
+
+@pytest.fixture
+def response_with_one_event():
+    with open(Path(__file__).parent / "fixtures/lippupiste_response.json", "r") as f:
```

Comment:
Alternatively:
```suggestion
def response_with_one_event(request):
    with open(request.path.parent / "fixtures/lippupiste_response.json", "r") as f:
```
Essentially the same thing (`request.path` returns a `Path` where the test was collected), so either one works. Wouldn't say that either one is better than the other, so do what you will.

Reply:
This removes `import Path from path` so it's a clear winner :crown: